samples/HttpRequestFeaturesv2/Startup.cs

```diff
@@ -0,0 +1,70 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.SpaServices.Webpack;
+using Microsoft.Azure.CognitiveServices.Personalizer;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+
+namespace HttpRequestFeaturesExample
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            string personalizerApiKey = Configuration.GetSection("PersonalizerApiKey").Value;
+            string personalizerEndpoint = Configuration.GetSection("PersonalizerConfiguration:PersonalizerEndpoint").Value;
```

Comment:
The readme.md says the endpoint in appsettings.json is "ServiceEndpoint" but the code says "PersonalizerEndpoint". One of these needs to change.